utils/collect.ts

```diff
@@ -136,9 +136,14 @@ const queryNominatim = async (lat: number, lon: number): Promise<NominatimRespon
 
         const nominatim = await queryNominatim(r.lat, r.lon);
 
+        let direction = parseInt(r.tags["camera:direction"]);
+        if (Number.isNaN(direction)) {
+            direction = parseInt(r.tags["direction"]);
+        }
         webcams.push({
             lat: r.lat,
             lon: r.lon,
+            direction: Number.isNaN(direction) ? 90 : direction, // point east by default
```

Comment:
referring to the comment on the type I prefer this to be undefined if it is not given